legate/core/runtime.py

```diff
@@ -1015,9 +1039,13 @@ def flush_scheduling_window(self) -> None:
         self._schedule(ops)
 
     def submit(self, op: Operation) -> None:
+        if op.can_raise_exception and self._precise_exception_trace:
```

Comment:
IIUC, the only computation we save if we don't do precise exceptions is that we skip a stacktrace save when launching an operation that can throw.

Given that this saving is at the python control program only (i.e. not on the critical computation path), and only affects operations that can throw, do you think the savings is worth the inaccuracy?

Reply:
The Python code is already a sequential bottleneck that limits the smallest problem size we can run, so I don't think we want to put any overhead on top of what we already have. Plus, the stacktrace generation is actually a pretty expensive operation, so I'd rather ask the users to mean it if they wanted it.